libcxx/test/configs/amdgpu-libc++-shared.cfg.in

```diff
@@ -0,0 +1,29 @@
+lit_config.load_config(config, '@CMAKE_CURRENT_BINARY_DIR@/cmake-bridge.cfg')
+
+config.substitutions.append(('%{flags}',
+  f'--target={config.target_triple} -Wno-multi-gpu -flto -mcpu=native'))
+config.substitutions.append(('%{compile_flags}',
+    '-nogpulib -fno-builtin-printf -nogpuinc -nostdlibinc '
+    '-I %{include-dir} -I %{target-include-dir}/../../ '
+    '-I %{target-include-dir} -I %{libcxx-dir}/test/support'
+))
+config.substitutions.append(('%{link_flags}',
+  '-O1 -nostdinc++ -nostdlib++ %{lib-dir}/crt1.o '
```

Comment:
```suggestion
  '-nostdlib++ %{lib-dir}/crt1.o '
```

`-nostdinc++` should be part of `compile_flags`, not `link_flags`.
`-O1` is controlled by a Lit parameter (`optimization=none|speed|size`), and shouldn't be hardcoded.

Reply:
None of these options pass `-O1` currently, `-Os` is `O2` with a few differences.

Reply:
Also does this pass it to the LTO job? Otherwise it will be O2.

Reply:
Okay I think AMDGPU actually recently fixed why I had to hard-code `-O1` on the link job (https://github.com/llvm/llvm-project/issues/64863). So I can probably omit this for now (Though it will possibly cause some more tests to fail I guess O0 is a good place to start).